packages/connect/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 9.1.12
+
+-   feat(connect-common): message channel to allow lazy handshake (79be923)
+-   chore(connect-ui): ui changes (dcb8e02, 03c6cc6, 936f6b5, 6b7ed69, 8f9f3e2, 4cc0cd6)
+-   fix(libs): changes in libs that should allow compiling connect with typescript option skipLibCheck: false
+-   chore(connect): split dev and prod builds. (535dd48)
+-   dependencies update: @trezor/blockchain-link, @trezor/blockchain-link-types, @trezor/type-utils, @trezor/analytics, @trezor/connect-common, @trezor/env-utils, @trezor/transport, @trezor/protobuf, @trezor/utxo-lib, @trezor/utils
```

Comment:
a link to changelog would be nice but problem is that not every package has changelog, for example blockchain-link has only its version bumped due to its internal dependnecies being updated. 
do we want to dedicate time to automate this better? @Hannsek @karliatto @martykan

Reply:
In general yes, I think it is worthy to automate as much as possible makes sense.

For connect I think we are already providing that as a comment in this PR, and since connect changelog we want to create manually something like that should be enough.

![image](https://github.com/trezor/trezor-suite/assets/5362163/4ada5b3c-9414-433f-982d-cfeb9704a2f7)

If we want to include it for every package then yes we need some updating in the automation. Do we want it in every package?

Reply:
I am not sure. it is to some extent also product decision.

Reply:
imho yes, why not, it gives the best transparency. it would take some effort to automate it but should be doable